docs/voldoen-aan-wetten-en-regels/hulpmiddelen/standaarden.md

```diff
@@ -0,0 +1,127 @@
+
+
+Voor een compleet overzicht van _internationaal_ gepubliceerde standaarden die relateren aan AI, [kijk op de website van de ISO](https://www.iso.org/committee/6794475/x/catalogue/p/1/u/0/w/0/d/0), of gebruik [NEN Connect](https://connect.nen.nl/Portal).
+
+## Wat moeten Nederlandse overheidsorganisaties doen?
+
+Nederlandse overheidsorganisaties kunnen standaarden gebruiken als praktische hulpmiddelen om te voldoen aan de eisen van de AI-verordening. 
+
+Hoe gebruik je standaarden?
+
+1. Kies de juiste standaarden: Begin met het identificeren van relevante standaarden en het begrijpen hoe deze toegepast kunnen worden. Focus op geharmoniseerde Europese normen die aansluiten bij de AI-verordening, en wanneer die er nog niet zijn, kies standaarden die aansluiten bij de onderwerpen uit de AI-verordening, door te kijken naar de lijst van 10 onderwerpen waarvoor een standaardisatieverzoek ligt (zie hierboven).
+3. Pas standaarden toe in de praktijk: Gebruik standaarden als richtlijnen om technische processen en organisatorische maatregelen in te richten, zoals het waarborgen van datakwaliteit, transparantie en betrouwbaarheid van AI-systemen​.
+4. Blijf op de hoogte: Volg de ontwikkelingen binnen JTC 21 en houd bij welke internationale standaarden worden opgenomen in het Europese normalisatieproces.
+Door standaarden actief te gebruiken, kunnen overheidsorganisaties hun AI-systemen veiliger en betrouwbaarder maken en tegelijkertijd voldoen aan de wettelijke vereisten van de AI-verordening.
+
+
+## Verwachte ontwikkelingen en tijdlijn
+De komende jaren zullen nieuwe AI-standaarden worden ontwikkeld en gepubliceerd om te voldoen aan de eisen van de AI-verordening. Deze standaarden zullen geharmoniseerd worden en juridisch relevant zijn, met name voor hoog-risico AI-systemen.
+Dit proces is complex en vereist samenwerking tussen vele stakeholders, inclusief industrie, overheid en maatschappelijke organisaties.
+
+Een overzicht van lopende en verwachte ontwikkelingen rondom standaarden voor de AI-verordening:
+
+- **2024 – 2025**: Europese standaardisatieorganisaties (CEN/CENELEC) werken aan de afronding van geharmoniseerde standaarden.
+Conceptversies van belangrijke horizontale standaarden verschijnen ter beoordeling door leden van de NEN normcommissie. (Meer info via [AI@nen.nl](mailto:AI@nen.nl))
+- **Begin 2025**: Publicatie van een update van het standaardisatieverzoek van de Europese Commissie.
+- **Eind 2025**: Publicatie van eerste geharmoniseerde Europese normen in het Official Journal of the EU.
+Deze normen creëren juridische zekerheid (Presumption of Conformity) voor organisaties die AI-systemen ontwikkelen.
+- **Augustus 2026**: De verplichtingen voor hoog-risico AI-systemen treden in werking. Vanaf dit moment moeten systemen voldoen aan de eisen van de AI-verordening, ondersteund door geharmoniseerde standaarden​.
+
+
+## De rol van JTC21
```

Comment:
In dit deel mis ik de verhouding tussen de JTC21, ISO en de NEN. Daar zouden wellicht nog twee zinnen aan kunnen worden besteed? Ik kan me herinneren dat ik deze afbeeldingen uit een presentatie van JTC21 heb gesniped. Ik vond dat super verhelderend. Wellicht hier toevoegen? Misschien heeft @FrieseWoudloper nog een goede visual van het proces of hoe zaken in elkaar grijpen? Ik zie overigens dat dit mogelijk al achter de bronvermelding schuil gaat, maar misschien een overweging waard om in deze tekst zaken visueel te maken?

![Standaardisatie 3](https://github.com/user-attachments/assets/3d15c95a-dcba-4d10-b535-a1d5d396d088)
![Standaardisatie](https://github.com/user-attachments/assets/e291bcb6-fdb6-4df9-b062-51b964bf2cf4)